ci-operator/config/heketi/heketi/master.yaml

```diff
@@ -0,0 +1,35 @@
+
+canonical_go_repository: github.com/heketi/heketi
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```

Comment:
Provide a separate image for your base if you need `python` and `tox` -- we can also build it from a Dockerfile in your repo if you'd like

Reply:
Do you think I should try to get that set up on the Heketi side first, or can we stick with this simpler approach to start with and then enhance the config later?

Reply:
I don't think the image you've selected has `python` or `tox` -- have you tested this locally?